app/src/main/java/org/covidwatch/android/exposurenotification/ExposureNotificationManager.kt

```diff
@@ -3,22 +3,23 @@ package org.covidwatch.android.exposurenotification
 import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration
 import com.google.android.gms.nearby.exposurenotification.ExposureNotificationClient
 import org.covidwatch.android.extension.await
+import org.covidwatch.android.extension.awaitNoResult
 import org.covidwatch.android.extension.awaitWithStatus
 import java.io.File
 
 class ExposureNotificationManager(
     private val exposureNotification: ExposureNotificationClient
 ) {
     /* API */
-    suspend fun start() = exposureNotification.start().awaitWithStatus()
+    suspend fun start() = exposureNotification.start().awaitNoResult()
```

Comment:
So this is just awaiting no value? Isn't it possible some other error might cause that too?

Reply:
No return value, yes. Meaning `Void`. This is because of bad Java -> Kotlin interoperability. EN methods return `Task<Void>` and Kotlin doesn't have `Void` type so it it's a mess there and that's is the solution that I found so far